docs/_scripts/update_preference_docs.py

```diff
@@ -14,7 +14,7 @@
 GUIDES_PATH = DOCS / "guides"
 IMAGES_PATH = DOCS / "images" / "_autogenerated"
 IMAGES_PATH.mkdir(parents=True, exist_ok=True)
-PREFERENCES_TEMPLATE = """(preferences)=
+PREFERENCES_TEMPLATE = """(napari-preferences)=
```

Comment:
Fixes the problem mentioned [here](https://github.com/napari/napari/issues/3654#issuecomment-973317892)

Reply:
just curious, did you ever verify whether this was still a problem if the preferences page actually gets built?  (I would assume that intersphinx would look for internal links before looking externally?